File: programmers/난이도별/level01.2016년/JiaLee0707.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

char* solution(int a, int b) {
    char* dayList[] = { "FRI", "SAT", "SUN", "MON", "TUE", "WED", "THU" };
    int month[12] = { 31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31 };
    char* answer = (char*)malloc(sizeof(char)*3);
    int count = 0;

    for (int i = 0; i < a - 1; i++) {
        count += month[i];
    }
    count += b - 1;

    answer = dayList[count%7];

    return answer;
}

File: programmers/난이도별/level01.문자열_내_p와_y의_개수/JiaLee0707.java
class Solution {
    boolean solution(String s) {
        boolean answer = true;
        int count = 0;
        
        for(int i=0; i<s.length(); i++) {
            char c = s.charAt(i);
            if(c == 'p' || c == 'P') count++;
            else if(c == 'y' || c == 'Y') count--;
        }
        if(count != 0) answer = false;
        
        return answer;
    }
}
